src/js/components/Barra/index.js

```diff
@@ -1,10 +1,10 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Row, Col, Table } from 'reactstrap';
+import { Row, Col } from 'reactstrap';
 import { Bar } from 'react-chartjs-2';
 import axios from 'axios';
 
-import * as Legenda from '../Legenda'
+import Legenda from '../Legenda';
```

Comment:
can we put this import right below `import Cartao from '../Cartao';`? just for we keep our components together